wp-content/themes/core/components/video.twig

```diff
@@ -5,8 +5,8 @@
 		{{ container_attrs }}
```

Comment:
It would be nice and should be doable to ensure that if there is a caption for the video, regardless of placement, to have it wrapped in the `<figure>` as a `<figcaption>`. I'll make a note of this for us to address in the near future, not a blocker to keeping this moving for now.

Reply:
Ok, not doing anything with this at this time in order to keep things moving forward